doc/api/net.md

```diff
@@ -161,6 +161,8 @@ when sockets were sent to forks.
 
 Callback should take two arguments `err` and `count`.
 
+Returns `server`.
```

Comment:
Elsewhere in the docs the return is listed as a bullet point along with the arguments. This should follow that convention.